packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/BarGraph/GoalLine/SimpleGoalLines.1.doc.mdx

```diff
@@ -4,7 +4,7 @@ import DataObject from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjec
 require('details-polyfill')
```

Comment:
Looks like you used different data object in example.

Reply:
fixed here: 9035efae5e927ade0858881c5f4fbeb2290e43b4